ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -96,6 +96,8 @@ export default class AlertsViewControls extends React.Component {
       });
     }
 
+    const hasMorePages = () => pageNums.length > 0 && count !== 1;
```

Comment:
@alexandru-io please cherry pick the commit from this [initial PR](https://github.com/mozilla/treeherder/pull/6050/commits) and then apply the other code changes, **in a separate commit**.